portal/src/main/webapp/js/api/cbioportal-datamanager.js

```diff
@@ -1399,15 +1393,15 @@ window.initDatamanager = function (genetic_profile_ids, oql_query, cancer_study_
 				    display_name: "Total mutations",
 				    is_patient_attribute: "0"
 				};
-				/*sample_clinical_attributes_set['NO_CONTEXT_MUTATION_SIGNATURE'] = {
+				sample_clinical_attributes_set['NO_CONTEXT_MUTATION_SIGNATURE'] = {
 				 attr_id: "NO_CONTEXT_MUTATION_SIGNATURE",
 				 datatype: "COUNTS_MAP",
 				 description: "Number of point mutations in the sample counted by different types of nucleotide changes.",
 				 display_name: "Nucleotide change of point mutations",
 				 is_patient_attribute: "0",
 				 categories: ["C>A", "C>G", "C>T", "T>A", "T>C", "T>G"],
```

Comment:
This is confusing, but I think the plus side is this decouples the display order from the order the data is served in.